modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -160,6 +160,8 @@ export class Tile3D {
     this.type = header.type;
     this.contentUrl = header.contentUrl;
 
+    this.traverser.options.maximumScreenSpaceError = tileset.options.maximumScreenSpaceError;
```

Comment:
To allow options to be undefined
```suggestion
    this.traverser.options.maximumScreenSpaceError = tileset.options?.maximumScreenSpaceError;
```